imap_processing/__init__.py

```diff
@@ -19,9 +19,13 @@
 # NOTE: Use a config dictionary, so it is a mutable global object,
 #       otherwise updating previous imports from other modules
 #       wouldn't have been updated globally (for example if referencing a string).
-config = {"DATA_DIR": Path(os.getenv("IMAP_DATA_DIR") or Path.cwd() / "imap-data")}
+config = {
+    "API_URL": "https://api.dev.imap-mission.com",
+    "DATA_DIR": Path(os.getenv("IMAP_DATA_DIR") or Path.cwd() / "imap-data"),
+}
 """imap_processing configuration dictionary.
 
+API_URL : This is the URL of the data archive API.
```

Comment:
During SDC tag-up, we were discussion about what to call this API and its endpoints. Many liked the name `data access API`. May be we should call that here too instead of `data archive API`?

Reply:
Sure, I can update the docstring to "data access API".

Should we also change this to `DATA_ACCESS_API` as the key? Or is `API`, `API_URL` (current), or something else short better? I'm open to suggestions here.

Reply:
My vote would be `DATA_ACCESS_API_URL` for consistency with what it is named ("Data Access API") and what the key is storing (a URL).

Reply:
Changed to DATA_ACCESS_API_URL now.